packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/e2e/pageObject.ts

```diff
@@ -0,0 +1,295 @@
+import type {Locator, Page, Request} from '@playwright/test';
+import {
+  isUaClickEvent,
+  isUaCustomEvent,
+} from '../../../../../../playwright/utils/requests';
+
+const minimumCitationTooltipDisplayDurationMs = 1500;
+
+export class GeneratedAnswerObject {
+  constructor(
+    public page: Page,
+    public streamId: string
+  ) {
+    this.page = page;
+    this.streamId = streamId;
+  }
+
+  get likeButton(): Locator {
+    return this.page.locator('[data-cy="feedback__like-button"]');
```

Comment:
If possible try to use the `getByRole` from Playwright, it is what's closer to "accessibility" testing we have for now.

Combine them with the `.getByRole().filter` to make specific Locators.

Reply:
improved 👍